php/circleci-decoupled/fpm/php-fpm.conf

```diff
@@ -0,0 +1,27 @@
+[global]
+error_log = /proc/self/fd/2
+
+[www]
+listen = [::]:${PHP_FPM_PORT}
+
+chdir = /data/api/app
```

Comment:
A lot of duplication here. Is this the only difference? Could this just be added as an included file?